client/src/components/Landing.style.js

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  background-color: var(--highlight);
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+
+export const H1 = styled.h1`
+  position: absolute;
+  width: 146px;
+  height: 60px;
+  left: 500px;
+  top: 180px;
+  font-family: Permanent Marker;
+  font-weight: bold;
+  font-size: 5rem;
+  transform: rotate(-7deg);
+  right: 10px;
+`;
+export const H2 = styled.h1`
```

Comment:
Calling it H2 when it is an H1 is a little bit confusing I think!

I would suggest calling this H1, and then we can make the one above into something like an H2